go/libkb/apiarg.go

```diff
@@ -11,6 +11,7 @@ const (
 	APISessionTypeNONE     APISessionType = 0
 	APISessionTypeOPTIONAL APISessionType = 1
 	APISessionTypeREQUIRED APISessionType = 2
+	APISessionTypeNONIST   APISessionType = 3
```

Comment:
what does NONIST mean and why is it necessary?

Reply:
this might not be necessary after the password flow is in place but I needed a away to have a NISTless session i tests since the pipeline checks the csrf token and the api lib use's a nist when available